cms/envs/common.py

```diff
@@ -991,6 +991,9 @@
 
     # This must be last so that it runs first in the process_response chain
     'openedx.core.djangoapps.site_configuration.middleware.SessionCookieDomainOverrideMiddleware',
+
+    # Check access for limited staff users.
+    'cms.djangoapps.contentstore.middleware.LimitedStaffAccessMiddleware'
```

Comment:
Please place this middleware before the one above it, since `SessionCookieDomainOverrideMiddleware` has the comment that `# This must be last so that it runs first in the process_response chain`.